student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('./person')
+
+class Teacher < Person
```

Comment:
- [ ] it looks like you by accident didn't change the name of this class to `Student`

Reply:
Yeah, I've forgotten, Thanks.